github/release_notes/util.py

```diff
@@ -376,6 +376,8 @@ def extract_release_notes(
     cn_current_repo -- component name of the current repository
     """
     release_notes = list()
+    if not text:
```

Comment:
when is this expected to happen? it seems to me as if it were preferrable to check this on the caller's side

Reply:
We saw issues when a pull request does not have a body, then text is None.

there are multiple callers (currently two) of this method, hence I added it to this central place